woocommerce/woocommerce-android
# Analytics Hub: Improve my store stats structure

### ⚠️ Depends on https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2630 and should be reviewed alongside it.

Changelog
==========
This PR syncs the My Store and Analytics Hub to use the same data source, enforcing the same data structure between them.

Disclaimer
==========
Since it will take a bunch of changes to achieve the complete scope, this PR is targeted to a feature branch called `improve/analytics-hub` that will aggregate everything, this way we can review piece by piece while keeping the production version intact.

How to Test
==========
1. Open the app and go to the My Store section. Verify that every single data between the My Store section and the Hub are in sync.

Update release notes:
